node/src/reactor/queue_kind.rs

```diff
@@ -4,7 +4,7 @@
 //! round-robin manner. This way, events are only competing for time within one queue, non-congested
 //! queues can always assume to be speedily processed.
 
-use std::{fmt::Display, num::NonZeroUsize};
+use std::fmt::Display;
```

Comment:
Why was this change made?  I think it was correct before the change.

Reply:
Fixed in 6cd7b1ee536c98cd9ccfb093df88cd18355678a1